jatin/postgres_assignment/src/controller/classcontroller.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+export { };
+
+const database = require('../services/classservices.ts');
+const validation = require('../helpers/validation_schema.ts');
+
+async function addClass(ctx: any) {
+  const obj = ctx.request.body;
+  try {
+    const reqBody = await validation.classSchema.validate(obj);
```

Comment:
This will not raise exception... Why is it inside try block ? same for all the try/catch blocks.